pino.js

```diff
@@ -273,6 +273,10 @@ Pino.prototype.child = function child (bindings) {
     throw new Error('missing bindings for child Pino')
   }
 
+  if (bindings.name && typeof bindings.name !== 'string') {
+    throw new Error('when provided, bindings.name must be a string for child Pino')
+  }
```

Comment:
definitely not needed, we use this without a name to set serializers and other things.